tests/ps_test.go

```diff
@@ -147,7 +147,9 @@ var _ = Describe("deis ps", func() {
 					afterProcs := scrapeProcs(app.Name, procsListing)
 
 					// compare the before and after sets of process names
-					Expect(len(afterProcs)).To(Equal(scaleTo))
+					if restart != "one" { // current workaround for an issue in deis/controller
```

Comment:
That is the only test that reliably failed for me when testing his PR
locally
On Apr 13, 2016 3:54 PM, "Matt Boersma" <notifications@github.com> wrote:

> In tests/ps_test.go
> <https://github.com/deis/workflow-e2e/pull/164#discussion_r59632849>:
>
> > @@ -147,7 +147,9 @@ var _ = Describe("deis ps", func() {
> >  					afterProcs := scrapeProcs(app.Name, procsListing)
> >
> >  					// compare the before and after sets of process names
> > -					Expect(len(afterProcs)).To(Equal(scaleTo))
> > +					if restart != "one" { // current workaround for an issue in deis/controller
>
> Is there an issue to track that we want to restore this test check in the
> future? And I'd add a TODO: or HACK: comment prefix to underline that
> idea.
>
> Also, e2e has passed without this change, so I wonder if it is truly
> needed now.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/deis/workflow-e2e/pull/164/files/ac70c952cb7e1fd6c86b4ef5b887a214a8233f3e#r59632849>
>